src/js/util/headlights.js

```diff
@@ -44,6 +45,7 @@ define(function (require, exports) {
      */
     var logEvent = function (category, subcategory, event) {
         if (global.debug) {
+            log.info("headlights: %s > %s > %s", category, subcategory, event);
```

Comment:
I think this should be removed before this is merged. I know it only happens in debug mode only, but I'm still interested in keep the console output readable for developers, which means trying to limit the output to that which is widely useful. I think, outside of testing this PR, this probably won't be widely useful.